### PR TITLE
DDI-428 update favicon

### PR DESCRIPTION
**Pull request by stephen-choi-amplitude**
# Amplitude Developer Docs PR

Update favicon to adjusts the blues used in our product

ref: https://amplitude.slack.com/archives/GDL32NVEH/p1681232628704989

## Description

In Safari
Screen Shot 2023-04-12 at 2 43 06 PM

In Chrome
Screen Shot 2023-04-12 at 2 43 00 PM

## Deadline

ASAP?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [x] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Comment by bgiori** [comment]
looks like your IDE reformatted the mkdocs file, can you revert that?

**Review by bgiori — CHANGES_REQUESTED** [review]
Dev docs already uses the new colors with a different favicon. What's the reason behind this change?

**Comment by stephen-choi-amplitude** [comment]
I didn't realize the old favicon was using the old blue color. I'll double check with Meredith.

**Comment by stephen-choi-amplitude** [comment]
Notes from my discussion with Meredith from the design team.

tl;dr; This is what we want(*)

- The main reason behind the change is to use our logo properly with correct color and design
- The A alone isn’t technically correct as our logo
- If there was a diff favicon that’s desired to be used for the dev center that’s certainly possible! 
- But if it reflects the logo it should use the sanctioned logo and not a diff color or inversion of it

**Review by justin-fiedler — APPROVED** [review]

**Comment by kevinpagtakhan** [comment]
@stephen-choi-amplitude do we still need this PR?

**Comment by markzegarelli** [comment]
Closing this in preparation of migrating the repository and inactivity
